erc20_moc/Erc20.noblock.js

```diff
@@ -86,15 +85,14 @@ class ERC20 {
    * @param amount {bigint}
    * @returns {Promise<void>}
    */
-  transferChain = async (dbClient, sender, receiver, amount) => {
+  transferChain = async (dbClient, sender, receiver, senderNamespace, receiverNamespace, amount, _symbol) => {
```

Comment:
why you need the override for symbol? something is wrong here. you transfer this.symbol and that must be used consistently